go/client/cmd_ctl_stop.go

```diff
@@ -36,6 +36,19 @@ func (s *CmdCtlStop) ParseArgv(ctx *cli.Context) error {
 }
 
 func (s *CmdCtlStop) Run() (err error) {
+	configCli, err := GetConfigClient(s.G())
+	if err != nil {
+		return err
+	}
+	config, err := configCli.GetConfig(context.TODO(), 0)
+	if err != nil {
+		return err
+	}
+
+	if config.ForkType == keybase1.ForkType_LAUNCHD {
+		return StopLaunchdService(s.G(), config.Label, true)
```

Comment:
Ah, so you did see the code in `service/ctl_osx.go`  What's the rationale for moving the code into the client?